src/neps/optimizers/info.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import annotations
+
+import os
+
+import yaml
+
+
+class SearcherConfigs:
+    """
+    This class provides methods to access default configuration details
+    for NePS optimizers.
+    """
+
+    @staticmethod
```

Comment:
Given the nature of the intended use of this class, would we prefer `@classmethod` or `@staticmethod` here?

Reply:
At the moment I left it as static class for its easy use-case. We can discuss further changes on this class.